src/pgen/torus.cpp

```diff
@@ -268,10 +405,20 @@ void ProblemModifier(ParameterInput *pin) {
            "\tThis translates to x1min, x1max = %g %g\n",
            std::exp(x1min), x1min, x1max);
   }
+
+  // Set Cv properly for radiation
+  const bool do_rad = pin->GetOrAddBoolean("physics", "rad", false);
+  if (do_rad) {
+    const std::string eos_type = pin->GetString("eos", "type");
+    if (eos_type == "IdealGas") {
+      const Real Gamma = pin->GetReal("eos", "Gamma");
+      const Real cv = (Gamma - 1.) * pc::kb / pc::mp;
+      pin->SetReal("eos", "Cv", cv);
+    }
+  }
```

Comment:
also not guaranteed to be what we want

Reply:
Yeah I mean Cv isn't going to be constant for the vast majority of EOS's we're going to run. I think we in general will need to solve for equal temperature value with a root find.

Reply:
This `torus` pgen already does do a rootfind to set the rad and gas to be at equal temperatures (`GasRadTemperatureResidual` at the top of this class). This segment of code setting `cv` just overrules whatever is in the input parameter file so you don't have to worry about correctly calculating `cv` in cgs units when setting that parameter file (previous phoebus inputs are scale-free MHD problems (or scale-free rad) so `cv` was always just set to `1` or something).

Reply:
Oh I see... Can we not just have some ability to set the units of `Cv` in the input file?

Reply:
well you would still have to enter some wonky number. I'm not sure what units would turn pc::kb/pc::mp into 1

Reply:
presumably in units of `mass = kb`. :P 

or alternatively we should have some kind of "scale free" mode?

Reply:
I'm not sure what the right solution is now. One could maybe have another option for setting scale-free eos that takes mean molecular weight and adiabatic index, say. I've made an issue so this can be deferred: #133